Parsers/ASimWebSession/Test/Cisco_Firepower_ASimWebSession_DataTest.csv

```diff
@@ -0,0 +1,12 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 21 records (100.0%) for field [EventProduct] of type [Enumerated]: [""Firepower""] (Schema:WebSession)"
+"(2) Info: Empty value in 14 records (66.67%)  in optional field [Url] (Schema:WebSession)"
```

Comment:
This is an mandatory field, cannot be empty. Please validate. Also, please help updating the Class to 'Mandatory' in tester csv

Reply:
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/e30d6ec1-329a-48e5-bbd6-cf32eccb7fe4)

Reply:
As discussed, we are not getting value in RequestUrl field in some of the data in our environment from source itself.
![image](https://github.com/Azure/Azure-Sentinel/assets/107753570/f162766f-f473-49f5-9711-a8a77fe6ac42)